src/components/ui/BaseButton.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <button v-if="!link" :class="mode">
```

Comment:
Class can be a string, you don't have to bind this.

Reply:
here I want to use mode as a prop and change class names dependly on the button type so I think that mode as props should be binded right?